pennylane/transforms/core/transform.py

```diff
@@ -91,18 +123,33 @@ def qnode_circuit(a):
                 qml.RZ(a, wires=1)
                 return qml.expval(qml.PauliZ(wires=0))
 
+    We apply the transform on our tranform in order to make it compatible with more PennyLane types.
+
     >>> dispatched_transform = transform(my_quantum_transform)
 
-    Now you can use the dispatched transform directly on qfunc and qnodes.
+    Now you can use the dispatched transform directly on :class:`pennylane.QNode`.
 
-    For QNodes, the dispatched transform populates the ``TransformProgram`` of your QNode. The transform and its
-    processing function are applied in the execution.
+    For :class:`pennylane.QNode`, the dispatched transform populates the ``TransformProgram`` of your QNode. The
+    transform and its processing function are applied in the execution.
 
-    >>> transformed_qnode = dispatched_transform(qfunc_circuit)
+    >>> transformed_qnode = dispatched_transform(qnode_circuit)
     <QNode: wires=2, device='default.qubit', interface='auto', diff_method='best'>
 
-    One subtlety here, this transform would not work for a qfunc because our transform return more than one case. If
-    it was not the case you would be able to dispatch on quantum functions.
+    >>> transformed_qnode.transform_program
+    TransformProgram(my_quantum_transform)
+
+    If we apply a second time the transform on the and :class:`pennylane.QNode`, we would add it to the transform
```

Comment:
Is there a word missing or an extra word here: 'on the and'?

Reply:
Thanks, fixed!